code/api/src/modules/user/queries.test.js

```diff
@@ -0,0 +1,86 @@
+import request from 'supertest'
+import express from 'express'
+import graphqlHTTP from 'express-graphql'
+import schema from '../../setup/schema'
+import models from '../../setup/models'
+import bcrypt from 'bcrypt';
+import db from '../../setup/database';
+
+describe('user queries',  () => {
+  let server;
+
+  beforeAll(() => {
+    server = express();
+
+    server.use(
+      '/',
+      graphqlHTTP({
+        schema: schema,
+        graphiql: false,
+      })
+    )
+  })
+
+  beforeEach(async() => {
+   const dracoMalfoy = {
+     id: 1,
+     name: "Draco Malfoy",
+     email: "malfoy@hogwarts.com",
+     password: bcrypt.hashSync('slytherin', 10),
+     role: "USER",
+     createdAt: new Date(),
+     updatedAt: new Date(),
+   };
+
+   const rubeusHagrid = {
+     id: 2,
+     name: "Rubeus Hagrid",
+     email: "grounds@hogwarts.com",
+     password: bcrypt.hashSync('buckbeak', 10),
+     role: "USER",
+     createdAt: new Date(),
+     updatedAt: new Date(),
+   };
```

Comment:
Instead of using the seeds for the test database, I am seeding the database here so that it can be easily wiped at the end of each round of tests.

Reply:
Nice Work!

Reply:
No one mentioned we were going with a Harry Potter convention for testing, my side is gonna need some updating!

Reply:
![](https://media1.giphy.com/media/B2m81PSTOiEjm/giphy.gif?cid=5a38a5a2fx72o0ubuknamm014kc061qi230obbwz2kza1uiy&rid=giphy.gif)